connectors/jdbc/translator-jdbc/src/main/java/org/teiid/translator/jdbc/DefaultSQLDialect.java

```diff
@@ -1,34 +1 @@
-/*
```

Comment:
The line endings here seem to have been mangled.

Reply:
IDE on windows.  Fixed and pushed.